AmplifyPlugins/Analytics/Podfile

```diff
@@ -7,22 +7,26 @@ pod 'SwiftFormat/CLI'
 pod 'SwiftLint'
 
 target 'AWSPinpointAnalyticsPlugin' do
-	pod 'Amplify', :path => '../../'
-  pod 'AWSPluginsCore', :path => '../../'
-  pod "AWSPinpoint", "~> #{AWS_SDK_VERSION}"
+    pod 'Amplify', :path => '../../'
+    pod 'AWSPluginsCore', :path => '../../'
+    pod "AWSPinpoint", "~> #{AWS_SDK_VERSION}"
 end
 
 target "HostApp" do
-  use_frameworks!
-  pod "CwlPreconditionTesting", :git => "https://github.com/mattgallagher/CwlPreconditionTesting.git", :tag => "1.2.0"
-  pod "CwlCatchException", :git => "https://github.com/mattgallagher/CwlCatchException.git", :tag => "1.2.0"
-  pod 'AmplifyTestCommon', :path => '../../'
+    use_frameworks!
+    pod "CwlPreconditionTesting", :git => "https://github.com/mattgallagher/CwlPreconditionTesting.git", :tag => "1.2.0"
+    pod "CwlCatchException", :git => "https://github.com/mattgallagher/CwlCatchException.git", :tag => "1.2.0"
+    pod 'AmplifyTestCommon', :path => '../../'
+    pod 'Amplify', :path => '../../'
+    pod 'AWSPluginsCore', :path => '../../'
+    
+    target "AWSPinpointAnalyticsPluginIntegrationTests" do
+        pod 'AmplifyPlugins/AWSAuthPlugin', :path => '../../'
```

Comment:
although there is no conflict to merge into master from this PR, this isn't going to with because the plugin has been renamed to `AWSCognitoAuthPlugin`

Reply:
to get the latest changes from remote
```
git fetch origin
git rebase origin/master
```
then you can update to latest. `pod 'AmplifyPlugins/AWSCognitoAuthPlugin',...`
and run pod install

Reply:
done